packages/android_intent/android/src/main/java/io/flutter/plugins/android_intent/AndroidIntentPlugin.java

```diff
@@ -0,0 +1,85 @@
+package io.flutter.plugins.android_intent;
```

Comment:
The paths need to be updated too - I'll put together a patch.  It's disconcerting that Travis didn't complain...